tests/settings/ConfigurationGatheringTest.py

```diff
@@ -21,32 +19,30 @@
 class ConfigurationGatheringTest(unittest.TestCase):
 
     def setUp(self):
-        self.log_printer = LogPrinter(NullPrinter())
-
         # Needed so coala doesn't error out
         self.min_args = ['-f', '*.java', '-b', 'JavaTestBear']
 
     def tearDown(self):
-        close_objects(self.log_printer)
+        close_objects()
 
     def test_gather_configuration(self):
-        args = (lambda *args: True, self.log_printer)
+        args = (lambda *args: True)
```

Comment:
args -> arg OR do `(lambda ..., )` with the trailing comma so it's a tuple. Your parentheses are redundant because there's no comma python will ignore them and that confuses readability.

So either we give this a proper name or we keep args and do the comma thing.